src/test/java/com/salesforce/storm/spout/dynamic/DynamicSpoutTest.java

```diff
@@ -1212,8 +1212,15 @@ private void validateNextTupleEmitsNothing(
         // Try a certain number of times
         final int originalSize = collector.getEmissions().size();
         for (int x = 0; x < numberOfAttempts; x++) {
+            // Call next Tuple
```

Comment:
This doesn't alter the behavior of the test, just provides better debug information when the test fails